zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java

```diff
@@ -339,6 +338,14 @@ private void broadcastAll(Message m) {
     }
   }
 
+  private void unicast(Message m, NotebookSocket conn) {
+    try {
+      conn.send(serializeMessage(m));
+    } catch (IOException e) {
+      LOG.error("socket error", e);
```

Comment:
@felixcheung I am using the approach used in the functions below:
  private void broadcast(String noteId, Message m) {
  private void broadcastExcept(String noteId, Message m, NotebookSocket exclude) {